stdlib/public/SDK/Dispatch/Time.swift

```diff
@@ -73,8 +72,12 @@ public struct DispatchWallTime : Comparable {
 }
 
 public func <(a: DispatchWallTime, b: DispatchWallTime) -> Bool {
-	if a.rawValue == ~0 || b.rawValue == ~0 { return false }
-	return -Int64(a.rawValue) < -Int64(b.rawValue)
+    if b.rawValue == ~0 {
```

Comment:
Oh weird, I have a Vim plugin that's supposed to auto-detect this sort of thing but apparently it doesn't work anymore >_<